packages/flutter/lib/src/widgets/spacer.dart

```diff
@@ -0,0 +1,67 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'basic.dart';
+import 'framework.dart';
+
+/// Spacer creates an adjustable, empty spacer that can be used to tune the
+/// spacing between widgets in a [Flexible] container, like [Row] or [Column].
+///
+/// Spacer takes an optional parameter [flex] that is used as the
+/// [Flexible.flex] value.
+///
```

Comment:
we should add some text here that points to mainAxisAlignment and talks about spaceAround, spaceBetween, etc.